docs/_data/mesheryctlcommands/cmds.yml

```diff
@@ -763,6 +763,18 @@ app:
               mesheryctl app view [application-name|application-id] -o json
           example:
               mesheryctl app view bookInfo -o json
+    import:
+      name: import
+      description: Import the app manifest into Meshery
+      usage: |
+        mesheryctl app import [flags]
+      flags:
+        file:
+          name: --file, -f
+          description: Path/URL to app file
+        source-type:
+          name: --source-type, -s
+          description: source type of application
```

Comment:
@alphaX86 what's the plan for having this automated?

Reply:
I have no qualms on making this automated but we would lose the desirability of the format we're having if we make this automated.

Through this tree format only, the index is being updated properly (even though it's manual process). If we want to make this automated, we want to either change the structure of index page (both YML and HTML page) or making a use of 3rd party tool. Maybe LLMs have the answer to this problem